### PR TITLE
Spark yarn

### PR DESCRIPTION
**Pull request by xqliang**
Add dependencies for spark-yarn

### CODE REVIEW

File: elasticsearch/src/main/java/zipkin/dependencies/elasticsearch/ElasticsearchDependenciesJob.java [hunk 1]
```diff
@@ -61,6 +61,9 @@ public static Builder builder() {
 
     Builder() {
       sparkProperties.put("spark.ui.enabled", "false");
+      // maybe support more spark properties?
+      sparkProperties.put("spark.hadoop.yarn.resourcemanager.address",
+              System.getProperty("spark.hadoop.yarn.resourcemanager.address", "0.0.0.0:8032"));
```
Comment (adriancole) [inline elasticsearch/src/main/java/zipkin/dependencies/elasticsearch/ElasticsearchDependenciesJob.java]: where does 0.0.0.8032 come from? Is this a default? if so, we should add comment of the documentation page or github link describing it.

Comment (xqliang) [reply elasticsearch/src/main/java/zipkin/dependencies/elasticsearch/ElasticsearchDependenciesJob.java]: it seems default to [${yarn.resourcemanager.hostname}:8032](https://hadoop.apache.org/docs/r2.7.4/hadoop-yarn/hadoop-yarn-common/yarn-default.xml) and `yarn.resourcemanager.hostname` is default to `0.0.0.0`.

File: main/pom.xml [hunk 2]
```diff
@@ -137,6 +137,31 @@
                     <exclude>META-INF/*.RSA</exclude>
                   </excludes>
                 </filter>
+                <filter>
+                  <!-- org.apache.xerces.jaxp.DocumentBuilderFactoryImpl used in spark-yarn -->
```
Comment (adriancole) [inline main/pom.xml]: this part wasn't yarn specific. fixed in master

Comment (xqliang) [reply main/pom.xml]: ok, removed

**Comment by adriancole** [comment]
thanks. let me know status of this.. ex if it works or not at the moment

**Comment by jorgheymans** [comment]
This seems stale, @xqliang are you still interested in this ?

**Comment by jorgheymans** [comment]
Closing for now, feel free to reactivate this PR if the need comes up again.

**Pull request closed without merging**